src/components/Export.tsx

```diff
@@ -21,10 +16,18 @@ import MenuItemSelect from "roamjs-components/components/MenuItemSelect";
 import { saveAs } from "file-saver";
 import getFullTreeByParentUid from "roamjs-components/queries/getFullTreeByParentUid";
 import getRoamUrl from "roamjs-components/dom/getRoamUrl";
-import { ExportDialogComponent } from "roamjs-components/types/query-builder";
 import getPageUidByPageTitle from "roamjs-components/queries/getPageUidByPageTitle";
 import getSubTree from "roamjs-components/util/getSubTree";
 import getShallowTreeByParentUid from "roamjs-components/queries/getShallowTreeByParentUid";
+import { ExportTypes } from "../utils/types";
+import { Result } from "roamjs-components/types/query-builder";
+
+type ExportDialogComponent = (props: {
+  onClose: () => void;
+  isOpen: boolean;
+  exportTypes: ExportTypes;
+  results: Result[] | ((isBackendEnabled: boolean) => Promise<Result[]>);
+}) => JSX.Element;
```

Comment:
A lot of this PR also includes inlining back the types from `roamjs-components/types/query-builder`. Only real changes are in `/src/index.ts`